src/wp-content/themes/twentytwelve/js/navigation.js

```diff
@@ -20,15 +20,25 @@
 		return;
 	}
 
+	// Assign an ID for the default page list if no menu is set as Primary.
+	if ( ! menu.id ) {
+		menu.setAttribute( 'id', 'twentytwelve-page-list-menu' );
```

Comment:
Apparently this works too 😊:
```suggestion
		menu.id = 'twentytwelve-page-list-menu';
```